packages/dnb-eufemia/src/extensions/forms/FieldBlock/FieldBlockDocs.ts

```diff
@@ -22,12 +22,12 @@ export const fieldBlockSharedProperties: PropertiesTableProps = {
     status: 'optional',
   },
   width: {
-    doc: '`small`, `medium`, `large`, `stretch` or `false` for predefined standard widths.',
+    doc: 'Will set the width for the whole block. Use `small`, `medium`, `large` for predefined standard widths. You can also set a custom width `{number}rem` or use `stretch` or `false`.',
```

Comment:
```suggestion
    doc: 'Will set the width for the whole block. Use `small`, `medium`, `large` for predefined standard widths. You can also set a custom width `{number}(rem|ch)` or use `stretch` or `false`.',
```

Don't have to write it that way, but we should mention `ch` too 🫡

Reply:
I removed the `ch` support. I have not managed to be useful, as it will always be off because of the padding, and for some reason, it was inconsistent when used few or many chars.